pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -222,6 +237,31 @@ func (f *framework) RunPrefilterPlugins(
 	return nil
 }
 
+// RunFilterPlugins runs the set of configured Filter plugins for pod on
+// the given node. If any of these plugins doesn't return "Success", the
+// given node is not suitable for running pod.
+// Meanwhile, the failure message and status are set for the given node.
+func (f *framework) RunFilterPlugins(pc *PluginContext,
+	pod *v1.Pod, nodeName string) *Status {
+
+	_, exist := f.nodeInfoSnapshot.NodeInfoMap[nodeName]
+	if !exist {
+		klog.Errorf("RunFilterPlugins: Fail to find NodeInfo in snapshot for name %s", nodeName)
+		return NewStatus(Error, "Fail to find NodeInfo in snapshot")
+	}
+
+	for _, p := range f.filterPlugins {
+		status := p.Filter(pc, pod, nodeName)
+		if !status.IsSuccess() {
+			klog.Errorf("RunFilterPlugins: error while running %s filter plugin for pod %s: %s",
```

Comment:
We should not log an error here. It is very common that a filter plugin may return "Unschedulable" for a given pod. We shouldn't log a scheduler error for all such instances.

Reply:
It has been addressed.